charts/prometheus-node-exporter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.0.1
 description: A Helm chart for prometheus node-exporter
 name: prometheus-node-exporter
-version: 1.15.0
+version: 1.16.0
```

Comment:
@DaviGia Need to bump to 1.16.1 before I merge this.

Reply:
Done